mozci/console/commands/decision.py

```diff
@@ -87,6 +87,7 @@ def create_task(self, push: Push) -> str:
                 f"docker-worker:cache:mozci-classifications-{environment}",
                 f"secrets:get:project/mozci/{environment}",
                 "notify:email:*",
+                "notify:matrix-room:!vNAdpBnFtfGfispLtR:mozilla.org",
```

Comment:
Maybe we should read this from the secret?

Reply:
@EvaBardou pointed out we have to update community-tc-config anyway, so no need for this.